builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -235,6 +247,7 @@ steps:
     TEST_SERVER: '$(testServer)'
     NODE_MODULES_PATH: '$(nodeModulesPath)'
     AZUREFUNCTIONS_SQLBINDINGS_TELEMETRY_OPTOUT: '1'
+    AZ_FUNC_TABLE_INFO_CACHE_TIMEOUT_MIN: '0'
```

Comment:
Let's not set this at the top level here so that other tests use the normal timeout. Instead pass it in the start function host call just for that specific test.